<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>livingcontrol</class>
 <widget class="QWidget" name="livingcontrol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListView" name="listView_program">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>171</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>驻场节目列表</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_device">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>171</width>
     <height>541</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>驻场设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文字采集</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_up">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>961</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_allow">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>770</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 0);
color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>允许</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_general">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>730</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重新生成</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_content">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>431</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_theme">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>470</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>450</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>主题</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_scene">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>510</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>场景</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_motion">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>570</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>情绪</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_para">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>690</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>参数设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>450</y>
     <width>361</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>识别中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>630</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>其他注释</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_head">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>650</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
